lib/commands/exec.js

```diff
@@ -32,7 +36,14 @@ module.exports = {
 		const log = require( '../utils/log' )();
 
 		return new Promise( ( resolve, reject ) => {
-			const newCwd = path.join( data.options.cwd, data.mgit.packages, data.name );
+			// The repository was not found so the package is skipped.
+			if ( !data.repository ) {
```

Comment:
> Moving it out to `lib/index.js`?

I found a solution which allows using current logs and does not execute the command. See: 799feda.